app/core/auth/__init__.py

```diff
@@ -88,15 +88,11 @@ async def wrapper(*args, **kwargs):
                 labels = []
 
             else:
-                result = supa.table('userTypes').select('''
-                        sources
-                        '''
-                    ).in_(
-                    'user_type', user_data.user.user_metadata.get('user_types')
-                    ).execute()
+                result = supa.table('userAttributes').select('id, user_id, user_type, "userTypes"(user_type, sources)').eq('user_id', user_data.user.id).execute()
                 labels = []
                 for data in result.data:
-                    labels.extend(data.get('sources'))
+                    labels.extend(data.get('userTypes', {}).get('sources', []))
+                print(f'{labels=}')
```

Comment:
Should this be log? `log` is already imported in the file it looks like.
```suggestion
                log.info(f'{labels=}')
```

Reply:
Ok yep, I'll change that

Reply:
Wow - I didn't know you could suggest a change and commit it right here in the conversation. That's awesome!